File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame19giugno2017/Negozio.java
public class Negozio {
    
    protected int codice, superficie;
    protected String indirizzo;
    protected String tipo;

    public Negozio(String tipo, int codice, String indirizzo, int superficie) {
        this.tipo = tipo;
        this.codice = codice;
        this.indirizzo = indirizzo;
        this.superficie = superficie;
    }

    public int getCodice() {
        return codice;
    }

    public int getSuperficie() {
        return superficie;
    }

    public String getTipo()
    {
        return tipo;
    }
}

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame19giugno2017/Python/negozio.py
class Negozio:
    
    codice: int
    superficie: int
    indirizzo: str
    tipo: str 
    
    def __init__(self, tipo: str, codice: int, indirizzo: str, superficie: int) -> None:
        self.tipo = tipo
        self.codice = codice
        self.indirizzo = indirizzo
        self.superficie = superficie

    def __str__(self) -> str:
        return str(self.codice) + "\t" + self.indirizzo + "\t" + str(self.superficie)